enterprise/models.py

```diff
@@ -843,13 +851,31 @@ class EnterpriseCustomerCatalog(TimeStampedModel):
             "included in the catalog."
         )
     )
+    enabled_course_modes = JSONField(
+        default=mode_slugs,
+        blank=True,
+        null=True,
+        load_kwargs={'object_pairs_hook': collections.OrderedDict},
+        help_text=_("Course modes that will be displayed on the Enterprise course enrollment landing page.")
```

Comment:
nit: `'Course modes that will be displayed on the enterprise course enrollment page.'`

Reply:
Change to `Ordered list of enrollment modes which can be displayed to learners for course runs in this catalog.`